pytest_kiwitcms/__init__.py

```diff
@@ -0,0 +1 @@
+from .plugin import *
```

Comment:
We need to figure out a naming scheme for the modules. In the TAP plugin I have `tap_plugin` but that is too generic. Maybe `tcms_xxx_plugin` will be good enough and I can adjust the other plugin. What do you think?

Reply:
I believe that `tcms_xxx_plugin` schema would work, so `tcms_pytest_plugin` sounds nice to me,

Reply:
Renamed the package, `plugin.py` is usual for pytest plugins, will contain all pytest-related hooks. Or you mean to rename this module?